bee-protocol/src/worker/solidifier/transaction.rs

```diff
@@ -35,15 +41,16 @@ impl TransactionSolidifierWorker {
 
     // TODO is the index even needed ? We request one milestone at a time ? No PriorityQueue ?
 
-    async fn solidify(&self, hash: Hash, index: u32) -> bool {
+    async fn solidify(&self, hash: Hash, index: MilestoneIndex) -> bool {
         let mut missing_hashes = HashSet::new();
 
-        tangle().walk_approvees_depth_first(
+        traversal::walk_approvees_dfs(
+            &tangle().inner,
             hash,
-            |_| {},
-            |vertex| !vertex.is_solid(),
-            |missing_hash| {
-                missing_hashes.insert(*missing_hash);
+            |_, v| v.get_meta().contains(Flags::SOLID),
+            |_, _| {},
+            |h| {
```

Comment:
Can you please revert to the old names ? Not a fan of single letter names :(

Reply:
Alright, T. :trollface: